azure/template.json

```diff
@@ -268,6 +274,15 @@
                     "storageAccountName": {
                         "value": "[variables('storageAccountName')]"
                     },
+                    "storageKind": {
+                        "value": "StorageV2"
+                    },
+                    "allowedIpAddressesList": {
+                        "value": "[parameters('storageAccountAllowedIPs')]"
+                    },
+                    "allowAzureServices": {
+                        "value": true
+                    },
```

Comment:
Can remove this

Reply:
"allowAzureServices": {
                        "value": true
                    },